### ISSUE TITLE
[vscode-apollo] Syntax highlighting for schema annotations not working properly depending on the annotation type

### ISSUE THREAD

**Issue by marcospgp** [apollographql/apollo-tooling#1042]
Multiline annotations get highlighted properly, while single line annotations don't.

![Screenshot](https://i.imgur.com/xhW5GsH.png)

**Comment by JakeDawkins** [comment]
I don't believe this way of commenting single-lines is [spec-compliant](https://github.com/facebook/graphql/blob/master/spec/Section%202%20--%20Language.md#comments) (although it does seem to work in apollo-server 🤔).

Since I can't seem to find this documented officially, I'd just change these single-line comments to use the `#` indicator like so 

```
  type User {
    # my comment here
    firstName: String
  }
```

**Comment by marcospgp** [comment]
You're confusing comments with annotations - comments are only visible in the source code, while annotations are used to document the schema and are visible on, for example, the graphql playground.

I could never find information on how to document the schema in any official vanilla graphql sources, but Apollo itself has [a section in the docs](https://www.apollographql.com/docs/apollo-server/essentials/schema.html#documentation) where they describe how you can document the schema.

The multi line annotations are highlighted properly by the extension, while single lines aren't.

**Comment by JakeDawkins** [comment]
Ah @marcospgp my apologies! I misunderstood this entirely when triaging. You're right! This should definitely be fixed 💯
